packages/core/src/bus/Brokers/RMQEventBus.spec.ts

```diff
@@ -0,0 +1,177 @@
+// eslint-disable-next-line max-classes-per-file
+import { bind, Command, Event, EventBus, EventHandler, RMQEventBus } from '..';
+import { Configuration } from '../../configuration';
+import { injectable } from 'tsyringe';
+
+describe('RMQEventBus', () => {
+  let config: Configuration;
+
+  beforeAll(() => {
+    config = new Configuration({
+      api: 'amqps://amq.development.playground.neuralegion.com:5672',
+      bus: 'EventBus',
+      clientQeueu: `agent:nnCF9MfHpbvdJVtSbQfKa1`,
+      connectTimeout: 10000,
+      heartbeatInterval: 5000,
+      reconnectTime: 5000,
+      credentials: {
+        username: 'bot',
+        token: '6vslx0u.nexp.jthl49f9urgdhf7y87vzxq0pt4jvk0mu'
+      }
```

Comment:
Oh... Never expose credentials in ~public~ repositories, especially with such permissions (as far as I can see, you have added all scopes). Please revoke your API key from the app asap. Moreover, it is not a unit test anymore, you should use mocks and spies to test functionality without the actual connection to RabbtiMQ.